### PR TITLE
 nextcloud27: 27.1.3 -> 27.1.4;  nextcloud26: 26.0.8 -> 26.0.9 

### PR DESCRIPTION
**Pull request by SuperSandro2000**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by dotlambda** [comment]
It would be nice if someone could fix the NixOS test.

**Comment by SuperSandro2000** [comment]
fixed that, too

File: nixos/tests/nextcloud/with-declarative-redis-and-secrets.nix [hunk 1]
```diff
@@ -62,6 +65,7 @@ in {
 
       services.postgresql = {
         enable = true;
+        package = pkgs.postgresql_14;
```
Comment (dotlambda) [inline nixos/tests/nextcloud/with-declarative-redis-and-secrets.nix:68]: Why is this needed?

Comment (SuperSandro2000) [reply nixos/tests/nextcloud/with-declarative-redis-and-secrets.nix:68]: otherwise the setup fails:

```
nextcloud # [    8.120489] systemd[1]: Started PostgreSQL Server.
nextcloud # [    8.124548] systemd[1]: Starting nextcloud-setup.service...
nextcloud # [    9.224207] postgres[999]: [999] WARNING:  no privileges could be revoked for "nextcloud"
nextcloud # [    9.302305] postgres[992]: [992] ERROR:  permission denied for schema public at character 14
nextcloud # [    9.303310] postgres[992]: [992] STATEMENT:  CREATE TABLE oc_migrations (app VARCHAR(255) NOT NULL, version VARCHAR(255) NOT NULL, PRIMARY KEY(app, version))
nextcloud # [    9.312801] nextcloud-setup-start[982]: Error while trying to initialise the database: An exception occurred while executing a query: SQLSTATE[42501]: Insufficient privilege: 7 ERROR:  permission denied for schema public
nextcloud # [    9.314932] nextcloud-setup-start[982]: LINE 1: CREATE TABLE oc_migrations (app VARCHAR(255) NOT NULL, versi...
nextcloud # [    9.316465] nextcloud-setup-start[982]:                      ^
nextcloud # [    9.319421] nextcloud-setup-start[982]: Trace: #0 /nix/store/ay71npxcw8gafabr4vaxsn6pkbdm5xmc-nextcloud-27.1.4/3rdparty/doctrine/dbal/src/Connection.php(1814): Doctrine\DBAL\Driver\API\PostgreSQL\ExceptionConverter->convert(Object(Doctrine\DBAL\Driver\PDO\Exception), Object(Doctrine\DBAL\Query))
```

**Comment by Ma27** [comment]
I thought the issue with postgresql got fixed already?

Anyways, happy to test & take a look when I'm back home.

**Review by dotlambda — APPROVED** [review]
